predictions/app/src/main/java/com/google/firebase/example/predictions/MainActivity.java

```diff
@@ -23,31 +24,34 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     public void configShowAds() {
-        long cacheExpiration = 60L;
-
         // [START pred_config_show_ads]
         final FirebaseRemoteConfig config = FirebaseRemoteConfig.getInstance();
 
-        Map remoteConfigDefaults = new HashMap<String, Object>();
+        Map<String, Object> remoteConfigDefaults = new HashMap<>();
```

Comment:
Note to reviewer: Made this change to keep it consistent with the other snippets.